Kernel/Arch/riscv64/CPU.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024, Leon Albrecht <leon.a@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Singleton.h>
+#include <Kernel/Arch/riscv64/CPU.h>
+#include <Kernel/Memory/MemoryManager.h>
+#include <LibDeviceTree/DeviceTree.h>
+#include <Userland/Libraries/LibDeviceTree/FlattenedDeviceTree.h>
+#include <Userland/Libraries/LibDeviceTree/Validation.h>
+
+static Singleton<OwnPtr<DeviceTree::DeviceTree>> s_device_tree;
+
+namespace Kernel {
+
+BootInfo s_boot_info;
+
+alignas(PAGE_SIZE) __attribute__((section(".bss.fdt"))) u8 s_fdt_storage[fdt_storage_size];
```

Comment:
Do we really need to align it to a page boundary?
An alignment of 8 bytes should be enough (this is what the devicetree spec wants)